src/ScriptBuilder/Subscription/Create_Oracle.sql

```diff
@@ -0,0 +1,29 @@
+﻿declare 
+  tableName varchar2(30) := UPPER(:1) || 'SS';
+  createTable varchar2(500);
+  n number(10);
+begin
+  select count(*) into n from user_tables where table_name = tableName;
+  if(n = 0)
+  then
+    
+    createTable :=
+       'CREATE TABLE ' || tableName || ' 
+        (
+          MESSAGETYPE NVARCHAR2(200) NOT NULL 
+        , SUBSCRIBER NVARCHAR2(200) NOT NULL 
+        , ENDPOINT VARCHAR2(200) NOT NULL 
+        , PERSISTENCEVERSION VARCHAR2(23) 
+        , CONSTRAINT ' || tableName || '_PK PRIMARY KEY 
+          (
+            MESSAGETYPE 
+          , SUBSCRIBER 
+          )
+          ENABLE 
+        ) 
+        ORGANIZATION INDEX';
```

Comment:
I admit I'm making assumptions based on my knowledge of the data model as it pertains to SQL Server and my understanding of SQL clustered indexes vs. Oracle index-organized tables. I don't know (yet) how to look at the execution plan but it's something we should definitely look at.